portfolio/src/main/webapp/script.js

```diff
@@ -33,28 +33,30 @@ function getComments() {
     .then(comments => {
       const commentsListElement = document.getElementById("comments-list");
       commentsListElement.innerHTML = "";
-      comments.forEach(line => {
-        commentsListElement.appendChild(createListElement(line));
-      });
+      comments.map(createListElement).forEach(line => commentsListElement.appendChild(line));
     });
 }
 
-/** Creates an <li> element containing comment data.*/
+/** Creates an <ul> element containing comment data.*/
 function createListElement(comment) {
-  const commentElement = document.createElement('li');
+  const commentElement = document.createElement('ul');
```

Comment:
Should it be `ul`? An HTML list is usually `ul` (**U**nordered **L**ist) or `ol` (**O**rdered **L**ist) and `li` the **L**ist **I**tems themselves.

Reply:
This question wasn't answered :)

Reply:
Ah, so sorry about that! I made it an unordered list so that the comments don't have bullet points.

Reply:
Ah, in that case I think it would've been better to create a `div` with a specific `class` rather than `ul`.

Reply:
Oh, that makes sense! I've made the adjustment and will include it in other PR. 👍